src/main/java/uk/gov/hmcts/reform/sscs/service/HmcHearingPartiesNotifiedApi.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sscs.config.FeignClientConfig;
+import uk.gov.hmcts.reform.sscs.model.partiesnotified.PartiesNotified;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@RestController
+@FeignClient(name = "hmc-hearing", url = "${hmc.url}", configuration = FeignClientConfig.class)
+public interface HmcHearingPartiesNotifiedApi {
+
+    String SERVICE_AUTHORIZATION = "ServiceAuthorization";
+
+    @PutMapping(value = "/partiesNotified", consumes = MediaType.APPLICATION_JSON_VALUE)
+    void updatePartiesNotifiedHearingRequest(
```

Comment:
Any contract tests for this?

Reply:
I think @JamieHMCTS has written contract test , I am focusing on feign client call ,  Once merge both code they will come.